README.md

```diff
@@ -73,6 +73,7 @@
 
 ## Deployment Tools
 
+* [bento](https://github.com/rapenne-s/bento/) - A KISS deployment tool to keep your NixOS fleet (servers & workstations) up to date.
```

Comment:
The tool itself looks pretty cool and continually developed, so its probably good to add.

As a nitpick, trim the last slash from the URL please.

More seriously, the given description doesn't do a great job of making it distinct from the other deployment tools. I'd recommend mentioning it's pull-based and that it leverages SSH.

Perhaps something like "A pull-based deployment agent to keep your NixOS fleet up-to-date over SSH."?